helpdesk_mgmt_crm/wizard/helpdesk_ticket_create_lead.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2022 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+import html2text
+
+from odoo import SUPERUSER_ID, _, api, fields, models
+
+
+class HelpdeskTicketCreateLead(models.TransientModel):
+    _name = "helpdesk.ticket.create.lead"
+    _description = "Helpdesk Ticket Create Lead"
+
+    ticket_id = fields.Many2one(
+        comodel_name="helpdesk.ticket",
+        required=True,
+        readonly=True,
+        domain=[("lead_id", "=", False)],
+    )
+    user_id = fields.Many2one(comodel_name="res.users")
+    team_id = fields.Many2one(comodel_name="crm.team")
+
+    @api.model
+    def default_get(self, fields):
+        vals = super().default_get(fields)
+        ticket = self.env["helpdesk.ticket"].browse([self.env.context.get("active_id")])
+        if ticket:
+            vals.update({"ticket_id": ticket.id})
+        return vals
+
+    def _prepare_vals(self):
+        return {
+            "ticket_id": self.ticket_id.id,
+            "name": self.ticket_id.name,
+            "partner_id": self.ticket_id.partner_id.id,
+            "user_id": self.user_id.id or self.ticket_id.user_id.id,
+            "team_id": self.team_id.id,
+            "description": html2text.html2text(self.ticket_id.description),
```

Comment:
The field is now HTML, so no need of this. Please put attention of these things, specially as this implies an extra library.

```suggestion
            "description": self.ticket_id.description,
```

Reply:
Thank you, I hadn't thought of that, I'll keep it in mind.